.github/workflows/analyze-project.yaml

```diff
@@ -38,3 +38,78 @@ jobs:
           pvsKey: ${{ secrets.PVS_KEY }}
           target: x64
           config: Debug
+
+  macos:
+    name: macOS 🍏 (clang-analyze)
+    runs-on: macos-14
+    # if: github.repository_owner == 'obsproject'
```

Comment:
I assume you meant to uncomment this before PRing

Reply:
Right, forgot about that one though I'm pondering whether we need to limit this anyway as `clang-analyze` is always available by default.